packages/yourloops/components/alarm/alarms-content-configuration.tsx

```diff
@@ -274,41 +159,57 @@ function AlarmsContentConfiguration(props: AlarmsContentConfigurationProps): JSX
                   id="low-bg-text-field-id"
                   value={lowBg.value}
                   error={lowBg.error}
+                  helperText={lowBg.error && (bgUnit === UNITS_TYPE.MGDL ? t('mandatory-integer') : t('mandatory-float'))}
                   type="number"
                   className={classes.textField}
                   variant="outlined"
                   size="small"
                   InputProps={{
                     inputProps: {
                       min: MIN_LOW_BG,
-                      max: MAX_LOW_BG
+                      max: MAX_LOW_BG,
+                      step: bgUnit === UNITS_TYPE.MGDL ? '1' : '0.1',
+                      'aria-label': t('low-bg-input')
+                    }
+                  }}
+                  FormHelperTextProps={{
+                    classes: {
+                      root: classes.inputHelperText
                     }
                   }}
                   onChange={(event) => onChange(+event.target.value, MIN_LOW_BG, MAX_LOW_BG, setLowBg)}
                   data-testid="low-bg-text-field-id"
                 />
-                <Typography>{t('mg/dL')}</Typography>
+                <Typography data-testid="bgUnits-minimum">{bgUnit}</Typography>
```

Comment:
This data-testid is not necessary (cf next comment in test, in which the data-testid `toto` is set on the parent box of this current element)